index.js

```diff
@@ -13,7 +13,19 @@ function BroccoliMergeTrees(inputNodes, options) {
     throw new Error('Expected array, got ' + inputNodes)
   }
   options = options || {}
-  Plugin.call(this, inputNodes, {
+  var newInputNodes = []
+  for (var i = 0 ; i < inputNodes.length ; i++ ) {
+    if (inputNodes[i].constructor.name  === this.constructor.name ) {
```

Comment:
@stefanpenner I can't think of any better option than checking constructor equality. Equality-checking the `.name` can definitely produce wrong results when you mix different versions of this plugin.